telemetry/common_fields.go

```diff
@@ -0,0 +1,32 @@
+package telemetry
+
+import (
+	"database/sql"
+
+	"github.com/status-im/telemetry/pkg/types"
+)
+
+func InsertCommonFields(db *sql.DB, data types.CommonFieldsAccessor) (int, error) {
```

Comment:
Why not just pass pointer to `CommonFields` here? I think using interface here is a bit overcomplicating.

```suggestion
func InsertCommonFields(db *sql.DB, data *types.CommonFields) (int, error) {
```

Reply:
Ah I didn't know it could work like this! much simpler, thanks